set.mm

```diff
@@ -639994,6 +640001,89 @@ number axioms (add ~ ax-10 , ~ ax-11 , ~ ax-13 , ~ ax-nul , and remove
     $( $j usage 'remulcand' avoids 'ax-mulcom'; $)
   $}
 
+  ${
+    $d a b x y $.
+    $( Lemma for ~ sn-mul02 . ` 0 x. _i = 0 ` .  (Contributed by SN,
+       30-Jun-2024.) $)
+    0tie0 $p |- ( 0 x. _i ) = 0 $=
```

Comment:
Why is this proof so long? The theorem could easliy be proven using ~mul02 and ~ax-icn. I think the usage of some axioms should be avoided here. Since this is a part of/lemma for ~sn-mul02, it should be named ~sn-mul02lem.
The label ~0tie0 should be used/reserved for the direct proof using ~mul02 and ~ax-icn.

Reply:
Since ~ 0tie0 is also used in ~ retinre (now ~ retire) i ended up renaming this to ~ sn-0tie0, a la ~ sn-it0e0